agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -113,15 +145,25 @@ def field_match_test(
 def auto_webhook_test(
     inputs: Dict[str, Any],
     output: str,
-    correct_answer: str,
+    data_point: Dict[str, Any],
     app_params: Dict[str, Any],
     settings_values: Dict[str, Any],
     lm_providers_keys: Dict[str, Any],
 ) -> Result:
     try:
+        correct_answer_key = settings_values.get("correct_answer_keys")
```

Comment:
is it singular or plural?
This is confusing `key = setting_values[keys]`
<img width="692" alt="CleanShot 2024-05-16 at 08 48 57@2x" src="https://github.com/Agenta-AI/agenta/assets/4510758/1c762ff7-60a6-459a-929a-409f0edfc65d">

Reply:
the settings will support multiple keys.
It depends on the evaluator if we would need a single or multiple correct_answer_keys
for ragas it would be correct_answer_keys = settings_values.get("correct_answer_keys")